src/webaudio.js

```diff
@@ -575,6 +575,17 @@ export default class WebAudio extends util.Observer {
         this.source.connect(this.analyser);
     }
 
+    /**
+     * @private
+     *
+     * Safari requires an explicit call to #resume before it will play back audio
```

Comment:
I would use 'some browsers' instead of Safari only.

Reply:
fixed.